dap-netcore.el

```diff
@@ -85,6 +85,24 @@ The slash is expected at the end."
     ("attach"
      (dap--put-if-absent conf :processId (string-to-number (read-string "Enter PID: " "2345"))))))
 
+(if (and (fboundp 'libxml-available-p)
```

Comment:
Why do you need this? Is the original `dap-netcore-download-url` link not working?
Having this code here will make a very perceivable delay when the user is loading `dap-netcore`, I don't really recommend it.

Reply:
Yes, original link is broken

Reply:
Also in modern emacs this code run asynchronously. I can't see any noticeable delay. But I use emacs 28 with native-compilation. Could someone test it on old emacs?

Reply:
In that case I think you should change the `dap-netcore-download-url` (both description and default value) to include `nil` as let `dap-netcore` to decide where to download. Then move these codes inside `dap-netcore--debugger-install` so that it will not affect loading time.

Reply:
Also, you should probably consider to use `lsp-download-install` in `dap-netcore--debugger-install` as well to prevent Emacs hangs while waiting for download

Reply:
@s-kostyaev

Reply:
Will see, thank you.

Reply:
done. @kiennq